tests/src/FunctionalJavascript/ContactSubmissionTest.php

```diff
@@ -107,8 +107,12 @@ public function testSelectContactElement() {
     }
     $this->getSession()->getPage()->selectFieldOption('Existing Contact', $this->contacts[1]['id']);
     $this->assertSession()->assertWaitOnAjaxRequest();
-    $this->getSession()->getPage()->fillField('First Name', 'Frederick');
-    $this->getSession()->getPage()->fillField('Last Name', 'Pabst');
+    $params = [
+      'first_name' => 'Frederick' . substr(sha1(rand()), 0, 7),
+      'last_name' => 'Pabst' . substr(sha1(rand()), 0, 7),
+    ];
+    $this->getSession()->getPage()->fillField('First Name', $params['first_name']);
```

Comment:
I'd favour that we remove all rand pre and post append from names. I'd rather see descriptive names.